frontend/package.json

```
@@ -37,5 +37,6 @@
       "last 1 firefox version",
       "last 1 safari version"
     ]
-  }
-}
+  },
+  "proxy": "http://localhost:1337"
```

```suggestion
  "proxy": "http://localhost:1338"
```
wrong port backend is on 1338. Assuming we want them to share the host and not separate origins.
I'm OK either way.